packages/store/src/-private/caches/identifier-cache.ts

```diff
@@ -39,11 +39,10 @@ import { hasId, hasLid, hasType } from './resource-utils';
 
 type ResourceData = unknown;
 
-const IDENTIFIERS = getOrSetGlobal('IDENTIFIERS', new Set());
```

Comment:
we always were checking the CACHE_OWNER symbol path first so this was just dead-perf